scripts/bagging_variables.sh

```diff
@@ -12,12 +12,14 @@ export BAG_ALWAYS="/odom /absodom /clock /tf /tf_static"
 # Define the directory that the bags will be stored in
 export BAG_DIR=~/bags
 
+# Which camera image type to use
+export CAM_SUFFIX="image_rect_color"
 
 # Topic variables that can be used from the bag command
-export bag_front_left_cam="/camera/front/left/camera_info /camera/front/left/image_raw"
-export bag_front_right_cam="/camera/front/right/camera_info /camera/front/right/image_raw"
+export bag_front_left_cam="/camera/front/left/camera_info /camera/front/left/$CAM_SUFFIX"
```

Comment:
Good idea, but that would be done in mil_common and probably best by @whispercoros